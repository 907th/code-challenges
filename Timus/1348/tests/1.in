8 -6 8 6
0 0 7
