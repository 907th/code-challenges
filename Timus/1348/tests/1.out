1.00
3.00
